.cirrus.yml

```diff
@@ -18,6 +18,8 @@ env:
 
   CCACHE_MAXSIZE: "250M"
 
+  CI_FOLDER: src/tools/ci
+
```

Comment:
This won't change anymore, so it can be removed now.

Reply:
Done.